ftdetect/rust.vim

```diff
@@ -1,7 +1,10 @@
 " vint: -ProhibitAutocmdWithNoGroup
 
 autocmd BufRead,BufNewFile *.rs call s:set_rust_filetype()
-autocmd BufRead,BufNewFile Cargo.toml setf FALLBACK cfg
+
+if v:version > 800 || v:version == 800 && has('patch613')
```

Comment:
You don’t need to use the old technique; `if has('patch-8.0.613')` will work correctly.

Reply:
:+1: I actually checked the docs, but apparently didn't read far enough: https://github.com/vim/vim/blob/v8.0.0613/runtime/doc/eval.txt#L8406-L8413